unit_scaling/functional.py

```diff
@@ -202,12 +204,20 @@ def layer_norm(
     add_args=[ternary_constraint_docstring],
 )
 def add(
-    input: Tensor,
-    other: Tensor,
+    input: Union[Tensor, int, float],
+    other: Union[Tensor, int, float],
     constraint: Optional[str] = "gmean",
     alpha: int = 1,
     out: Optional[Tensor] = None,
 ) -> Tensor:
+    # Adding a constant shouldn't change scale
+    if (
+        isinstance(input, (int, float))
+        or isinstance(other, (int, float))
+        or input.numel() == 1
+        or other.numel() == 1
```

Comment:
Ah good point, was too busy thinking about the output std. I've tweaked this so that scalar inputs still keep the output std unchanged, but now handle broadcasting for grads correctly